libconglomerate.user.js

```diff
@@ -1278,11 +1278,11 @@ function getInvItem(item) {
 
 function parseSafeItems() {
     var items = [], len, i, j, safestatus, safeType, safeOptions, componentMatch, componentId, component, count, alchemyItems;
-    var safe = document.evaluate("//form[@name='footlockergrab']", document, null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null);
+    var safe = document.evaluate("//form[@name="footlockergrab"][input[@value="retrieve"]]", document, null, XPathResult.ORDERED_NODE_SNAPSHOT_TYPE, null);
```

Comment:
The double quotes are a syntax error. Single (') vs. double quote (") is a style choice and it's not entirely consistent in libC, but it's largely single quotes with double quotes if the string contains a single quote (like here).
I'm guessing you pasted in an XPath query which you had single-quoted, and you used the double quotes inside it?

Also, are there any footlockergrab forms that *don't* include a retrieve input?
It's fine either way, I'm curious about it.

Reply:
Oh goodness, the syntax error is a terrible oversight and typo on my part, it’s correct on my test copy.

As for the input the only one currently that is something other than retrieve is the one causing the issue, that one has a value of discard. I’m not certain if redundancy there is required?

Reply:
> As for the input the only one currently that is something other than retrieve is the one causing the issue, that one has a value of discard. I’m not certain if redundancy there is required?

That sounds good then. You exclude it early, but also the "redundancy" isn't quite just redundancy, because in the future other select tags *could* be added.
The trade off of specifying the select's name attribute is that if they change it it'll break, but I think that's much less likely than them adding a new tag.